rules/dotnet-upgrade-for-complex-projects/rule.md

```diff
@@ -99,61 +99,74 @@ There are several ways to migrate project from ASP.NET to ASP.NET Core. We stron
 
 ### Create side-by-side incremental project with [.NET Upgrade Assistant](https://dotnet.microsoft.com/en-us/platform/upgrade-assistant)
 
-This will create a new .NET 8 project. For functionalities that have not yet been migrated, YARP will redirect them to the .NET Framework web application.
+After you've [installed the .NET Upgrade Assistant extension](https://learn.microsoft.com/en-au/dotnet/core/porting/upgrade-assistant-install#install-the-visual-studio-extension),
+right-click on the project in the Solution Explorer window, and select Upgrade.
 
-### Configure YARP
+![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3699937/3303daaf-0dea-4b34-9f59-53fd55acf2ef)
+
+A tab is opened which provides, based on your project type, different styles of upgrade:
+
+- In-place project upgrade
+  This option upgrades your project without making a copy.
+
+- Side-by-side project upgrade
+  Copies your project and upgrades the copy, leaving your original project alone.
+
+- Side-by-side incremental
+  A good choice for complicated web apps. Upgrading from ASP.NET to ASP.NET Core requires quite a bit of work and at times manual refactoring. This mode puts a .NET project next to your existing .NET Framework project, and routes endpoints that are implemented in the .NET project, while all other calls are sent to .NET Framework application.
+
+  This mode lets you slowly upgrade your ASP.NET or Library app piece-by-piece.
+
+On more complex projects you might find that Upgrade Assistant only provides you with the side-by-side incremental option. That is also the option we will be focusing on.

```

Comment:
Reword sentence to not include "we" - use the 3rd person perspective as described [here](https://my.sugarlearning.com/SSW/items/9369/how-to-write-ssw-rules).

Reply:
```suggestion
On more complex projects you might find that Upgrade Assistant only provides you with the side-by-side incremental option. That is also the option that is covered here.
```